components/Nav/MobileNavbar.js

```diff
@@ -0,0 +1,154 @@
+import React, { PureComponent } from 'react'
+import styled, { css } from 'styled-components'
+
+import rem from '../../utils/rem'
+import { navbarHeight } from '../../utils/sizes'
+import { paleGrey } from '../../utils/colors'
+import { mobile } from '../../utils/media'
+import Link from '../Link'
+import NavLinks from './NavLinks'
+import Social from './Social'
+// import Search from './Search'
+import Logo from './Logo'
+import NavSeparator from './NavSeparator'
+import NavButton from './NavButton'
+
+const Wrapper = styled.div`
+  display: none;
+
+  ${mobile(css`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: ${rem(navbarHeight)};
+  `)}
+`
+
+const SecondaryMenu = styled.div`
+  position: absolute;
+  top: ${rem(navbarHeight)};
+  left: 0;
+  right: 0;
+
+  ${p => p.open ? css`
+    height: ${rem(navbarHeight)};
+  ` : css`
+    height: 0;
+  `}
+
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 ${rem(20)};
+  transition: height 0.1s;
+
+  -webkit-user-select: none;
+  -webkit-overflow-scrolling: touch;
+  overflow-x: scroll;
+  overflow-y: hidden;
+
+  background: ${paleGrey};
+  color: #868686;
+`
+
+const LogoLink = styled(Link).attrs({
+  unstyled: true,
+  href: '/',
+})`
+  display: inline-block;
+  vertical-align: center;
+  margin-right: ${rem(35)};
+`
+
+const ArrowWrapper = styled.div`
+  transition: transform 0.1s;
+
+  ${p => p.rotate && css`
+    transform-origin: 50% 55%;
+    transform: rotate(180deg);
+  `}
+`
+
+const PaddedSocial = styled(Social)`
```

Comment:
I'm not too happy about wrapping the `Social` component here, since it's our own :/

It goes against the container -> structural -> presentational components order

Reply:
You're totally right. I didn't think of this before `container -> structural -> presentational` but it makes sense for me, too.